src/tests/ftest/util/server_utils.py

```diff
@@ -319,6 +319,31 @@ def stop_server(setname=None, hosts=None):
     subprocess.call(["pkill", "^ssh$"])
 
 
+def clean_server(hosts, scm=False, mountpoint="/mnt/daos"):
+    """Clean the tmpfs  on the servers.
+
+    Args:
+        hosts (list): list of host names where servers are running
+        scm (bool): if true, remove tmpfs mount
+        mountpoint (str): mount point path to clean
+    """
+    clean_cmds = [
+        "find {} -mindepth 1 -maxdepth 1 -print0 | \
+        xargs -0r rm -rf".format(mountpoint)
+    ]
+
+    if scm:
+        clean_cmds.append(
+            "sudo umount {0}; sudo rm -rf {0}".format(mountpoint))
```

Comment:
This looks like a pretty big and dangerous hammer.  Surely we'd only want to remove a (n empty) directory that the device was mounted on, yes?  IOW, if the directory the device was mounted on was not empty, we really have no business emptying it.

```suggestion
            "sudo umount {0}; sudo rmdir {0}".format(mountpoint))
```